tests/v3_api/test_kontainer_drivers.py

```diff
@@ -0,0 +1,88 @@
+import sys
+import os
+import time
+from .common import wait_for_condition, get_admin_client, validate_cluster, cluster_cleanup, random_test_name, find_condition, wait_for
+
+CREDENTIALS = os.environ.get('RANCHER_GKE_CREDENTIAL', "")
+GKE_MASTER_VERSION = os.environ.get('RANCHER_GKE_MASTER_VERSION', "")
+
+
+def test_create_cluster_with_kontainer_driver():
+    os = sys.platform
+
+    client = get_admin_client()
+    driver = client.create_kontainer_driver({
+        "active": True,
+        "baseType": "kontainerDriver",
+        "builtIn": False,
+        "url": "https://github.com/nathan-jenan-rancher/kontainer-engine-example-driver/releases/download/updategke/kontainer-engine-driver-example-" + sys.platform,
+        "createDynamicSchema": True
+    })
+
+    driver = client.reload(driver)
+
+    wait_for_condition(
+        client, driver,
+        lambda x: find_condition(x, 'Active') == "True",
+        lambda x: 'Condition is: ' + find_condition(x, 'Active'))
+
+    driver = client.reload(driver)
```

Comment:
save the driver off `wait_for_condition` as it returns the resource

Reply:
Done.